### ISSUE TITLE
Consider making email/country NOT required in QuickSubmit

### ISSUE THREAD

**Issue by stranack** [pkp/pkp-lib#3316]
QuickSubmit is often used to upload back issues, where author emails and country are unavailable or unknown. Journal Managers frequently put fake content into those fields to proceed, which is bad practice.

We should consider not making those two fields required.

**Comment by asmecher** [comment]
Partial dupe of https://github.com/pkp/pkp-lib/issues/1864 -- could you summarize the requirements for both in a single entry, @stranack?

**Comment by t4x0n** [comment]
I agree with this requirement, especially with the email, the country is typical in affiliation in affiliation information, even in old publication but the email is different because a lot of journals only publish correspondence author email.

**Comment by asmecher** [comment]
Closed in favour of https://github.com/pkp/pkp-lib/issues/3561
